javaslang/src-gen/main/java/javaslang/API.java

```diff
@@ -84,6 +90,865 @@ private API() {
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new Vector instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> Vector<T> vector(T... elements) {
+      return Vector.of(elements);
+    }
+
+    /**
+     * Alias for {@link List#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new List instance containing the given element
+     */
+    public static <T> List<T> list(T element) {
+      return List.of(element);
+    }
+
+    /**
+     * Alias for {@link List#of(Object...)}
+     *
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new List instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> List<T> list(T... elements) {
+      return List.of(elements);
+    }
+
+    /**
+     * Alias for {@link Stream#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new Stream instance containing the given element
+     */
+    public static <T> Stream<T> stream(T element) {
+      return Stream.of(element);
+    }
+
+    /**
+     * Alias for {@link Stream#of(Object...)}
+     *
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new Stream instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> Stream<T> stream(T... elements) {
+      return Stream.of(elements);
+    }
+
+    /**
+     * Alias for {@link Queue#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new Queue instance containing the given element
+     */
+    public static <T> Queue<T> queue(T element) {
+      return Queue.of(element);
+    }
+
+    /**
+     * Alias for {@link Queue#of(Object...)}
+     *
+     * @param <T>      Component type of elements.
+     * @param elements Zero of more elements.
+     * @return A new Queue instance containing the given elements
+     * @throws NullPointerException if {@code elements} is null
+     */
+    @SuppressWarnings("varargs")
+    @SafeVarargs
+    public static <T> Queue<T> queue(T... elements) {
+      return Queue.of(elements);
+    }
+
+    /**
+     * Alias for {@link LinkedHashSet#of(Object)}
+     *
+     * @param <T>     Component type of element.
+     * @param element An element.
+     * @return A new LinkedHashSet instance containing the given element
+     */
+    public static <T> LinkedHashSet<T> linkedHashSet(T element) {
```

Comment:
Our names are currently inconsistent, see #1488. In Scala we use either interface names or class names for object creation but never names of non-existing types.

Examples:

```scala
scala> Seq(1, 2, 3)
res0: Seq[Int] = List(1, 2, 3)

scala> Set(1, 2, 3)
res1: scala.collection.immutable.Set[Int] = Set(1, 2, 3)

scala> scala.collection.immutable.SortedSet.empty[String] + ("a", "c", "b")
res3: scala.collection.immutable.SortedSet[String] = TreeSet(a, b, c)
```

These are all interfaces. The is no such thing like `LinkedSet`. The name would be inconsistent.